maquetado/setup/scss/tech.scss

```diff
@@ -23,6 +23,13 @@
   }
 }
 
+.tech-container:hover {
```

Comment:
This could be `.tech-container:hover::before`

Also, add a `transition` attribute to `tech-container` for transition to work as expected